design_docs/parallel_tasks.md

```diff
@@ -0,0 +1,77 @@
+<h1> Title: Support Parallel Tasks <br>
+Xylar Asay-Davis <br>
+date: 2017/02/22 <br>
+</h1>
+<h2> Summary </h2>
+Currently, the full analysis suite includes 7 tasks, 5 for the ocean and 2 for sea ice.
+The number of tasks is expected to grow over time.  Task parallelism in some
+form is needed to allow as many tasks as desired to be run simultaneiously.
+Successful completion of this design will mean that the analysis suite produces
+identical results to the current `develop` branch but that several analysis
+tasks (a number selected by the user) run simultaneously.
+
+<h1> Requirements </h1>
+
+<h2> Requirement: Tasks run simultaneously <br>
+Date last modified: 2017/02/22 <br>
+Contributors: Xylar Asay-Davis
+</h2>
+There must be a mechanism for running more than one analysis task simultaneously.
+
+<h2> Requirement: Select maximum number of tasks <br>
+Date last modified: 2017/02/22 <br>
+Contributors: Xylar Asay-Davis
+</h2>
+There must be a mechanism for the user to select the maximum number of tasks
+to run simultaneously.  This might be necessary to control the number of processors
+or the amount of memory used on a given machine or (in the case of running
+analysis on login nodes) to be nice to other users on a shared resource.
```

Comment:
I have thoughts on implementation.  Yes, I think it will be a config option.  But given that we're at the requirements stage, I'm not worrying about design yet.